docs/usage/design.rst

```diff
@@ -0,0 +1,26 @@
+Design of SCAutolib
+=============================
+
+This section describes design decisions that were made during
+implementation with their explanations.
+
+* **SCAutolib uses MVC pattern adopted for CLI and test usage.**
+
+  MCV pattern is introduced to improve organization and structure of SCAutolib.
+  Further design decisions respects this pattern giving developers some baseline
+  for structuring of code. MVC is adopted into the context of SCAutolib in the
+  following way:
+  * Models are all basic classes in the library (CAs, Users, Card, etc.) that
+    provides low-level atomic functionality
+  * View is a CLI interface for manual interaction with the library
+  * Controller is a heart of the library that handles  logic behind both
```

Comment:
Please remove double space.

Reply:
how do you mean "remove"? double spaces define indent for the lists
Check how does it looks like after rendering https://scautolib.readthedocs.io/en/latest/usage/design.html

Reply:
I mean "...handles **'space' 'space'** logic..."